test/benchmarks.cpp

```diff
@@ -153,150 +156,150 @@ void benchmarkLoopVariable(std::function<void(QInterfacePtr, int)> fn, bitLenInt
 void benchmarkLoop(std::function<void(QInterfacePtr, int)> fn, bool resetRandomPerm = true,
     bool hadamardRandomBits = false, bool logNormal = false)
 {
-    benchmarkLoopVariable(fn, MaxQubits, resetRandomPerm, hadamardRandomBits, logNormal);
+    benchmarkLoopVariable(fn, max_qubits, resetRandomPerm, hadamardRandomBits, logNormal);
```

Comment:
Normalize between mxQbts and max_qubits for naming

Reply:
See my comment about snake versus camel, above.